lib/job/index.ts

```diff
@@ -67,8 +93,10 @@ class Job {
     this.attrs = {
       ...attrs,
       // NOTE: What is the difference between 'once' here and 'single' in agenda/index.js?
-      type: type || 'once',
-      nextRunAt: nextRunAt || new Date()
+      name: attrs.name ?? '',
+      priority: attrs.priority,
+      type: type ?? 'once',
+      nextRunAt: nextRunAt ?? new Date()
```

Comment:
?? instead of || intended?

Reply:
it's intended here, as we check for "undefined" not "null".
I will check type definitions that 'null' is invalid.